lib/livebook_web/live/hooks/sidebar_hook.ex

```diff
@@ -42,6 +43,10 @@ defmodule LivebookWeb.SidebarHook do
      |> put_flash(:error, error)}
   end
 
+  defp handle_info({:default_hub_changed, _hub_id}, socket) do
```

Comment:
Sorry, I think there was some confusion. I don't think we need to show it as default in the sidebar. it is too much information. I would only show it as a default in the team_component page itself. So you can remove the changes to this module and the `default_hub={@default_hub}` argument in the .layout :)

Reply:
To be clear, i was thinking about showing the "default" label in the same place we show the "livebook teams" label in `teams_component`.